NotebookAnalyzers/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import pynblint
+import linters
+
+app = FastAPI()
+nb_linter = linters.NbLinter()
+repo_linter = linters.RepoLinter()
+
+
+@app.get('/')
+def index():
+    return {'data': {'message': 'Welcome to the pynblint API'}}
+
+
+@app.get('/linters')
+def linters_descr():
+    return {"data": [{"id": nb_linter.id, "description": nb_linter.description}, {"id": repo_linter.id, "description": repo_linter.description}]}
```

Comment:
Potremmo inserire gli oggetti di classe *Linter in una lista e interare su questa lista per popolare il valore di "data". Ad es.:

```suggestion
    return {"data": [{"id": linter.id, "description": linter.description} for linter in linters_list]}
```

In questo modo, se dovessimo aggiungere linter in futuro non dovremmo preoccuparci di modificare questa funzione.

Reply:
Guardando l'implementazione che hai proposto in #39, direi che un dizionario sia la struttura dati ideale, piuttosto che una lista. Lo snippet che ti ho suggerito diventa:

```python
return {"data": [{"id": linter_id, "description": linters_dict[linter_id].description} for linter_id in linters_dict]}
```